sdk/core/Azure.Core/aotcompatibility/AotCompatibilityTestApp.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net7.0</TargetFramework>
```

Comment:
This will be a pain to keep up to date every year. Maybe we need a `LatestTargetFramework` in the root _Directory.Build.props_ or something or, rather, in one of the eng/ imports so it's centrally managed.

Reply:
You should be able to copy a lot of what's here to get the benefits of the repo's shared MSBuild infra: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/tests/public/Azure.Core.Tests.Public.csproj#L1

Reply:
I'm going to delete this test project here but will capture this feedback for the CI pipeline I'm working on now.